website/search/util.py

```diff
@@ -93,3 +103,13 @@ def create_atom_feed(name, data, query, size, start, url, to_atom):
         feed.add(**to_atom(doc))
 
     return feed
+
+
+def illegal_unicode_replace(atom_element):
+    """ Replace an illegal for XML unicode character with nothing.
+    This fix thanks to Matt Harper from his blog post:
+    https://maxharp3r.wordpress.com/2008/05/15/pythons-minidom-xml-and-illegal-unicode-characters/
+    """
+    if atom_element:
+        return re.sub(RE_XML_ILLEGAL_COMPILED, '', atom_element)
```

Comment:
Since you have a compiled regex, you can use the `sub` *method* of the regex object rather than the `re.sub` *function*. I think you had this before, but you were not handling `None` input.